src/main/java/com/kitaab/hisaab/ledger/controller/ReminderController.java

```diff
@@ -0,0 +1,55 @@
+package com.kitaab.hisaab.ledger.controller;
+
+import com.kitaab.hisaab.ledger.dto.request.reminder.SendReminderRequest;
+import com.kitaab.hisaab.ledger.dto.response.SuccessResponse;
+import com.kitaab.hisaab.ledger.service.ReminderService;
+import lombok.AllArgsConstructor;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+
+import static com.kitaab.hisaab.ledger.constants.ApplicationConstants.RECEIVED;
+import static com.kitaab.hisaab.ledger.constants.ApplicationConstants.REMINDER_ID;
+import static com.kitaab.hisaab.ledger.constants.ApplicationConstants.SENT;
+
+@RestController
+@RequestMapping("/api/reminder")
```

Comment:
move `/api/reminder` to ApplicationConstants

Reply:
done